src/Lean/Server/Watchdog.lean

```diff
@@ -132,13 +135,8 @@ section FileWorker
   def stdout (fw : FileWorker) : FS.Stream :=
     FS.Stream.ofHandle fw.proc.stdout
 
-  def readMessage (fw : FileWorker) : IO JsonRpc.Message := do
-    let msg ← fw.stdout.readLspMessage
-    if let Message.response id _ := msg then
```

Comment:
I've inlined this function into `forwardMessages` because the erase must happen when the message is sent to the client, not when it's received from the worker.  That is, this function must not be called anywhere else except from `forwardMessages`.